packages/adapter-cloudflare-workers/files/entry.js

```diff
@@ -1,3 +1,4 @@
+/* global GENERATE_NONCES */
```

Comment:
would it work to remove this line and do `globalThis.GENERATE_NONCES` instead?

Reply:
Maybe, I'll try it.